lib/help.js

```diff
@@ -45,12 +45,19 @@ exports.customHelp = function(generator, usageFilePath, cmd) {
   }
 
   // Append USAGE file if any
-  var exists = g.f(usageFilePath);
-  if (exists) {
-    helpStr.push(exists);
+  var usageText = g.f(usageFilePath);
+  if (usageText !== usageFilePath) {
+    helpStr.push(usageText);
   }
 
   helpStr = helpStr.join('\n');
+
+  if (command === 'loopback-cli') {
+    return helpStr
+      .replace(/ (yo|slc) loopback:/g, ' lb ')
+      .replace(/ (yo|slc) loopback/g, ' lb');
```

Comment:
> .replace(/ (yo|slc) loopback:?/g, ' lb'); will handle both cases

Notice that in the second case, we don't want space after `lb`. Sure, I can craft a more complex regexp, but I think it's not worth it. As a popular joke goes, "I wrote a regexp to solve my problem. Now I have two problems." 😉